### DISCUSSION TITLE
Priority of Around Advices for a given Method

### DISCUSSION THREAD

**Discussion by subash89** [micronaut-projects/micronaut-core#8917] [General] [answered]
Suppose I have a method which have three custom around advices specified. I need a specific  advice to get invoked first. How can I specify such order or priority ?

**Comment by graemerocher** [comment, marked as answer]
with either the `@Order` annotation on each interceptor or implementing `Ordered`. Also see https://docs.micronaut.io/latest/api/io/micronaut/aop/InterceptPhase.html which provides some useful constants you can use for the priority value for fitting your interceptor between existing implementations
